doc/md/tutorial-grpc-edges.md

```diff
@@ -110,10 +110,10 @@ import (
 
 	_ "github.com/mattn/go-sqlite3"
 
-	"github.com/rotemtam/ent-grpc-example/ent/category"
-	"github.com/rotemtam/ent-grpc-example/ent/enttest"
-	"github.com/rotemtam/ent-grpc-example/ent/proto/entpb"
-	"github.com/rotemtam/ent-grpc-example/ent/user"
+	"ent-grpc-example/ent/category"
+	"ent-grpc-example/ent/enttest"
+	"ent-grpc-example/ent/proto/entpb"
+	"ent-grpc-example/ent/user"
```

Comment:
Can you elaborate? I see the module is named differently here: https://github.com/rotemtam/ent-grpc-example/blob/main/go.mod#LL1C8-L1C44

Reply:
The tutorial is not meant to import a remote package.

The code starts from a locally written folder from

```
mkdir ent-grpc-example
cd ent-grpc-example
go mod init ent-grpc-example
```

And the rest of the tutorial use `ent-grpc-example`, not `github.com/rotemtam/ent-grpc-example`.

You can easily see such inconsistency in by searching for it: https://github.com/ent/ent/search?q=ent-grpc-example.